examples/tinkerbell/adding-a-user-to-tinkerbelltemplateconfiguration.md

```diff
@@ -0,0 +1,26 @@
+# Add a user to the `TinkerbellTempalteConfiguration`
+
+The `create-user` action creates a user in the root file system called `tinkerbell` with password `tinkerbell`.
+
+```
+apiVersion: anywhere.eks.amazonaws.com/v1alpha1
+kind: TinkerbellTemplateConfig
+metadata:
+  name: cpd
+spec:
+  template:
+    global_timeout: 6000
+    id: ""
+    name: cpd
+    tasks:
+    - actions:
+      // Append to existing actions.
+      - environment:
+          BLOCK_DEVICE: /dev/sda2
```

Comment:
put a comment next to this saying they would need to change BLOCK_DEVICE based on their disk type. Maybe should also mention that our Ubuntu images have UEFI as the first partition, so OS would be on the second.

Reply:
Updated